wizeline/remix-project-lab
# Members form

# What does this PR do?

- [Display roles and skills on members table of the view project page](https://github.com/wizeline/remix-project-lab/commit/92d6618e143a897146c11959e2d9f1121f592e03)
- [chore: changes to prettier defaults](https://github.com/wizeline/remix-project-lab/commit/e5199a9a47e28f02ff520f39af61855ef79eb458)
- [feat: remove project members field from ProjectForm, add own members route to edit them](https://github.com/wizeline/remix-project-lab/commit/c5c6872e52841ebc5472bb00a7e7a919da1a52e5)


# Where should the reviewer start?

- app/routes/projects/$projectId.members.tsx
- app/models/project.server.ts
  - `updateProjects` function
  - `createProject` function just adds owner as a project member
  - `updateProject` no longer updates also projectMembers

# How should this be manually tested?

1. Create a new project, notice only the owner gets added as a project member
2. You can now edit just project members clicking on the edit link from the members table

# Any background context you want to provide?



# What are the relevant tickets?



# Screenshots

![Screen Shot 2022-12-09 at 1 00 56](https://user-images.githubusercontent.com/15214/206644263-f8508ad2-773d-4a4e-ab1c-21a3da57f518.png)

![image](https://user-images.githubusercontent.com/15214/206644428-a6e76f55-fb10-4284-94dc-c8b545afe68f.png)


# Questions



# Checklist
